### PR TITLE
[Edge WS ] Approach & Scope

### PR DESCRIPTION
**Pull request by ASawwaf**
@bfcohen @walterkozlowski @Karimrabie @peterwoerndle 

Can you have look at this

### CODE REVIEW

**Comment by ASawwaf** [comment]
@bfcohen @walterkozlowski @Karimrabie @peterwoerndle

any comment to merge

File: doc/tech/usecases.md [hunk 1]
```diff
@@ -11,3 +11,26 @@
...
+Edge Approach:
+-	 All gaps under Reference Model , Reference Architecture 01 -OpenStack- , Reference Architecture 02 - K8s will be identified and Edge requirements will be fulfilled  under Reference Model , Reference Architecture 01 -OpenStack- , Reference Architecture 02 - K8s  
+
+Edge scope under CNTT will cover :
+
+-	Define Edge locations based on use case 
+-	Define some guideline about external factors that can affect the edge like WAN latency based on telco use case 
+-	Define VNF/CNF profiles ( if needed )specific to Edge use case such as profile required acceleration technologies 
+-	Define resources requirements in terms of compute , Storage and networking , new concept can be introduced such as hyper coverage infrastructure 
+-	Define Different Architecture model as no size fill fit all
```
Comment (pgoyal01) [inline doc/tech/usecases.md:26]: Suggest change:
"Define Different Architecture model as no size fill fit all" --> "Define different Architecture models as no one size fits all"

File: doc/tech/usecases.md [hunk 2]
```diff
@@ -11,3 +11,26 @@
 
 <a name="5.1"></a>
 ## 5.1 Edge
+
+3-Approach & Scope  : 
+
+Edge Approach:
+-	 All gaps under Reference Model , Reference Architecture 01 -OpenStack- , Reference Architecture 02 - K8s will be identified and Edge requirements will be fulfilled  under Reference Model , Reference Architecture 01 -OpenStack- , Reference Architecture 02 - K8s  
+
+Edge scope under CNTT will cover :
```
Comment (pgoyal01) [inline doc/tech/usecases.md:20]: Should the scope cover "Scale" -- edge locations are likely to be in thousands even hundreds of thousands and that has implications.

File: doc/tech/usecases.md [hunk 3]
```diff
@@ -11,3 +11,26 @@
...
+
+-	Define Edge locations based on use case 
+-	Define some guideline about external factors that can affect the edge like WAN latency based on telco use case 
+-	Define VNF/CNF profiles ( if needed )specific to Edge use case such as profile required acceleration technologies
```
Comment (pgoyal01) [inline doc/tech/usecases.md:24]: minor typo corrections:

"Define VNF/CNF profiles (if needed) specific to Edge use cases such as profile required for acceleration technologies

File: doc/tech/usecases.md [hunk 4]
```diff
@@ -11,3 +11,26 @@
...
+
+Edge Approach:
+-	 All gaps under Reference Model , Reference Architecture 01 -OpenStack- , Reference Architecture 02 - K8s will be identified and Edge requirements will be fulfilled  under Reference Model , Reference Architecture 01 -OpenStack- , Reference Architecture 02 - K8s  
+
+Edge scope under CNTT will cover :
+
+-	Define Edge locations based on use case 
+-	Define some guideline about external factors that can affect the edge like WAN latency based on telco use case 
+-	Define VNF/CNF profiles ( if needed )specific to Edge use case such as profile required acceleration technologies 
+-	Define resources requirements in terms of compute , Storage and networking , new concept can be introduced such as hyper coverage infrastructure
```
Comment (pgoyal01) [inline doc/tech/usecases.md:25]: minor typos:

"Define resource requirements in terms of compute, storage and networking; new concepts, such as hyper converged infrastructure, can also be introduced"

File: doc/tech/usecases.md [hunk 5]
```diff
@@ -11,3 +11,26 @@
...
+-	Define resources requirements in terms of compute , Storage and networking , new concept can be introduced such as hyper coverage infrastructure 
+-	Define Different Architecture model as no size fill fit all 
+-	
+
+Out of Scope 
+
+-	The API that can be exposed to 3rd party application developer 
+-	The Architecture of VNF/CNF ( CNTT Guideline only )
+-	Exact location of Edge deployment may be vary from Operator to another 
+-	Larceny may be vary based on operator transmission type ( Microwave , Fixed ) , a well Transmission network reachability and architecture ( Backhaul , front haul , Mid haul )
```
Comment (pgoyal01) [inline doc/tech/usecases.md:34]: is there a typo or "larceny" (theft) is the correct word
Sentence needs changing

File: doc/tech/usecases.md [hunk 6]
```diff
@@ -11,3 +11,26 @@
...
+-	Onboarding of VNF/CNF on edges cloud infrastructure 
+
```
Comment (pgoyal01) [inline doc/tech/usecases.md:36]: General typos:
- space before commas
-space after/before parentheses open and close

File: doc/tech/usecases.md [hunk 7]
```diff
@@ -11,3 +11,26 @@
...
+-	Define VNF/CNF profiles ( if needed )specific to Edge use case such as profile required acceleration technologies 
+-	Define resources requirements in terms of compute , Storage and networking , new concept can be introduced such as hyper coverage infrastructure 
+-	Define Different Architecture model as no size fill fit all 
+-	
+
+Out of Scope 
+
+-	The API that can be exposed to 3rd party application developer 
+-	The Architecture of VNF/CNF ( CNTT Guideline only )
+-	Exact location of Edge deployment may be vary from Operator to another
```
Comment (pgoyal01) [inline doc/tech/usecases.md:33]: I am missing what is really intended by this bullet under "out of scope" -- obviously locations will vary by operator

Typo:
"Exact location of Edge deployments may vary from Operator to Operator"

or 
"Exact location of Edge deployment will vary by Operator"

Comment (ASawwaf) [reply doc/tech/usecases.md:33]: @pgoyal01 . I am trying to contain the scope of CNTT edge

**Comment by rabi-abdel** [comment]
moving PR content to https://github.com/cntt-n/CNTT/pull/1449 to avoid confusion

**Pull request closed without merging**